### ISSUE TITLE
[Orbit] 01-sin-plus-quad: Propose f(x) = sin(x) + 0.1*x^2 by inspection (symmetric quadratic envelope + odd sinusoid)

### ISSUE THREAD

**Issue by wwang2** [wwang2/bench-toy-symreg-1776493842#2]
Strategy: Propose f(x) = sin(x) + 0.1*x^2 by inspection (symmetric quadratic envelope + odd sinusoid)
Parents: none (root)

**Branch:** [orbit/01-sin-plus-quad](https://github.com/wwang2/bench-toy-symreg-1776493842/tree/orbit/01-sin-plus-quad)
**Code:** [solution.py](https://github.com/wwang2/bench-toy-symreg-1776493842/blob/orbit/01-sin-plus-quad/orbits/01-sin-plus-quad/solution.py)
**Log:** [log.md](https://github.com/wwang2/bench-toy-symreg-1776493842/blob/orbit/01-sin-plus-quad/orbits/01-sin-plus-quad/log.md)

**Comment by wwang2** [comment]
## Eval Check
**Result:** VERIFIED
**Measured:** 0.000000 (MSE)
**Seeds:** 3/3 — all seeds return 0.0 (test set is deterministic, σ_test=0)
**Evaluator:** eval-v1
**Target met:** yes (target < 0.01)

**Branch:** [orbit/01-sin-plus-quad](https://github.com/wwang2/bench-toy-symreg-1776493842/tree/orbit/01-sin-plus-quad)
**Solution:** [solution.py](https://github.com/wwang2/bench-toy-symreg-1776493842/blob/orbit/01-sin-plus-quad/orbits/01-sin-plus-quad/solution.py)

**Comment by wwang2** [comment]
**orbit-reviewer:** orbit/01-sin-plus-quad — Advisory Notes

### Code quality

The code in `solution.py` is clean and minimal:

```python
def f(x: np.ndarray) -> np.ndarray:
    x = np.asarray(x, dtype=float)
    return np.sin(x) + 0.1 * x**2
```

- Only `numpy` is imported. No `sklearn`, no `scipy.optimize`, no fitting loops. Constraint satisfied.
- `solution.py` does NOT import or read `evaluator.py` or `generate_data.py`. No spirit-of-cheating concern.
- The docstring in `solution.py` explicitly restates the parity-decomposition reasoning, consistent with `log.md`.
- The formula in the code (`np.sin(x) + 0.1 * x**2`) exactly matches the hypothesis stated in `log.md` (`f(x) = sin(x) + 0.1 * x^2`).
- `make_figures.py` does import `generate_data` — appropriate for figure generation only; the evaluator-facing `solution.py` is clean.

[CODE NOTE] `make_figures.py` inserts `research/eval` onto `sys.path` to import `generate_data`. This is intentional and appropriate (needed to reproduce figures from committed data), but worth noting it is the figure script, not the solution itself.

No code issues found.

### Significance

- **Verified metric:** MSE = 0.000000 (confirmed by eval-check, 3/3 seeds)
- **Baseline (zero model):** MSE = 1.0 (from `config.yaml` `baselines.zero`)
- **delta baseline:** −1.000000 absolute (100% relative improvement)
- **delta best:** this is the root orbit; no prior best to compare against
- **Target:** MSE < 0.01 — beaten by the full floating-point gap
- **min_relative_improvement threshold:** 0.1 (10%) — exceeded by a factor of ~10×
- **Noise floor:** test data is deterministic (no noise, `seed=99` fixed, `generate_test_data` uses `np.linspace` with no RNG). Seed variance is σ_test = 0 by construction. A "vs 2σ" comparison does not apply in the usual sense — the zero residual is exact, not a sampling fluctuation.

[SIGNIFICANCE] metric=0.000000, δ=−1.000000 (100%), vs threshold 0.1 → **Significant** (exact symbolic match; noise floor is 0 on clean test set)

Note on the [SUSPICIOUS IMPROVEMENT] heuristic: the 100% relative improvement would normally trigger a flag, but this is expected and explained — the orbit proposes the exact generator function, the test set is clean, and the problem was explicitly designed for this outcome. No metric gaming.

### Reproducibility

- Deterministic seed: test data uses fixed `seed=99` inside `generate_data.py::generate_test_data`; train data uses fixed `seed=42`. Both are deterministic.
- `eval_version: eval-v1` recorded in `log.md` frontmatter. ✓
- Seeds documented in `log.md` body: yes — the log explicitly states "seed=99 fixed inside `generate_data.py::generate_test_data`". ✓
- Hyperparameters: there are none (closed form, no tunable parameters beyond the symbolic coefficients, which are exact). ✓
- `make_figures.py` is committed and fully reproducible from `solution.py` + `generate_data.py` + `train_data.csv`/`test_data.csv`. ✓

No reproducibility issues found.

### Figures

![results](https://raw.githubusercontent.com/wwang2/bench-toy-symreg-1776493842/refs/heads/orbit/01-sin-plus-quad/orbits/01-sin-plus-quad/figures/results.png)

![narrative](https://raw.githubusercontent.com/wwang2/bench-toy-symreg-1776493842/refs/heads/orbit/01-sin-plus-quad/orbits/01-sin-plus-quad/figures/narrative.png)

**results.png** — train scatter + test target + closed-form fit overlay. Legible, axes labeled, legend present and not overlapping data. Dashed orange overlays the solid blue target exactly (as expected). Title reports MSE = 0.000e+00. Style conforms to `research/style.md` (muted palette, no top/right spines, constrained_layout). Single panel — appropriate for this simple result.

[FIGURE NOTE] `results.png`: the legend sits in the upper center and is visually clear; however, since the orange dashed line and blue solid line are nearly coincident, a reader might not immediately see both. The legend entry "overlays target exactly" covers this well — no change needed, just noted.

**narrative.png** — two-panel figure: (a) training residuals vs x, (b) parity decomposition. Both panels use `(a)` / `(b)` labels added both in subplot titles and via `ax.text`. Style conforms. Colors are consistent with palette defined in `make_figures.py`.

[FIGURE ISSUE] `narrative.png`: panel (a) has a panel label `(a)` duplicated — it appears both in the subplot title string `"(a) Training residuals ..."` AND as a bold `ax.text` annotation at `(-0.12, 1.05)`. This produces visual redundancy (the bold `(a)` floats above the title that already starts with "(a)"). Same for panel (b). Suggest: remove the prefix `(a)` / `(b)` from the title strings and keep only the `ax.text` label, or vice versa.

[FIGURE ISSUE] `narrative.png`: the `fig.suptitle` uses `y=1.04` which places it above the panel-label `ax.text` at `y=1.05` in axes-fraction coordinates. On the rendered PNG the suptitle and the floating bold `(b)` are very close and may collide depending on backend. Suggest: use `y=1.08` for suptitle or remove panel prefixes from title strings.

[FIGURE ISSUE] `narrative.png`: the noise-band `fill_between` in panel (a) uses `[-4.2, 4.2]` as x-extent but `ax.set_xlim(-4.2, 4.2)` — this is consistent, but the band label in the legend (±σ_noise = 0.03) references the declared noise level rather than the empirically measured residual std (0.028). Minor inconsistency; consider adding the empirical std to the band label or the legend.

No missing figures. The two-figure layout (results + narrative) tells the complete setup → result → verification story.

### Quality hints

- **References section:** `log.md` has no `## References` section. This orbit uses no external papers (closed-form by inspection), so this is acceptable — but worth noting for future orbits that do cite prior work.
- **Prior Art & Novelty:** Present and appropriately hedged. The "no novelty claim" stance is honest and correct for an infrastructure-validation orbit. ✓
- **Punchline clarity:** The first two lines of the Result/Hypothesis section are immediately clear: the hypothesis is stated, then verified. ✓
- **Glossary:** Present (`## Glossary`). All abbreviations used in the log (MSE, σ) are defined there. ✓
- **Prose hygiene:** The derivation section leads with intuition (parity split observation) before the math. ✓

[QUALITY NOTE] `log.md` has no `## References` section. Acceptable here (no papers cited), but the section is recommended by convention. Add an empty `## References` block or a note "No external references — derivation by inspection" to make the absence explicit.

[QUALITY NOTE] DPI in `make_figures.py` is set to 200. `research/style.md` specifies "150 for research logs, 300 for publication/Issue embeds." 200 is between the two recommendations. The figures are used as Issue embeds, so 300 would be more conformant. Minor.

---
*Advisory only — no verdict gate. Findings consumed by campaign-reviewer at milestone.*
